rhel6/guide/services/dhcp/dhcp_server_configuration/group.yml

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-title: 'Disable DHCP Server'
+title: 'DHCP Server Configuration'
```

Comment:
See above comment

Reply:
What above comment are you referring to here? This is for DHCP server configuration, which iirc, is not done in `/etc/network-scripts/**ifcfg**`?

Reply:
@cipherboy sorry. Copy/paste error. Ignore.